src/js/AppHeader.js

```diff
@@ -15,30 +15,15 @@ import { Link } from 'react-router-dom';
 import uiController from './Controllers/UIController'
 import ScrollableMessage from './ScrollableMessage';
 
-import {ReactComponent as IconMenu} from '../img/icons/icon-menu.svg'
-import {ReactComponent as IconCart} from '../img/icons/icon-cart.svg'
-import {ReactComponent as PermissionsIcon} from '../img/static/0x49.svg'
+import {ReactComponent as IconMenu} from '../img/static/0x49.svg'
 
-class AppStoreIcon extends React.Component {
+class MainMenuSettings extends React.Component {
     render() {
-        return (<div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
-                <Link to="/permissionapplist" style={{ marginRight: 10, marginLeft: 'auto' }}>
-                    <div className="app-icon">
-                        <div className="static-icon">
-                            <div className="svg-wrap">
-                                <PermissionsIcon/>
-                            </div>
-                        </div>
-                    </div>
-                </Link>
-                <Link to="/appstore" style={{ marginRight: 0 }}>
-                    <div className="app-icon">
-                        <div className="static-icon">
-                            <div className="svg-wrap">
-                                <IconCart/>
-                            </div>
-                        </div>
-                    </div>
+        return (<div>
+                <Link to="/settings">
+                    <span className="svg-wrap settings-menu">
```

Comment:
```suggestion
                    <span className="settings-menu">
```
Also I would remove this class, this class makes it so the icon is automatically filled in, which I don't feel is necessary in this case.